Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/ExamControllerTest.java

```diff
@@ -544,6 +542,10 @@ void testGetStudentExamProgressOverview() throws Exception {
         assertEquals(secondQuestion.getProblemTitle(), otherProblem.getTitle());
     }
 
+    @DisplayName("Give a exam, two questions in the exam " +
+            "When delete the other_problem And get the examOverview include two questionViews " +
+            "Should the exam and examOverView are the same " +
+            "And firstQuestion should succeed, secondQuestion should not found the problem")
```

Comment:
Give an exam with two questions A, B, and the problem related to B has been deleted,
When get exam overview, question A should have problem title and question B should be notFound.